lib/alp_workloads/kvm_workload_utils.pm

```diff
@@ -58,6 +59,16 @@ sub set_kvm_container_image {
     }
 }
 
+sub set_host_bridge_interface_with_host {
```

Comment:
How about moving the definition of this function to https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/virt_autotest/utils.pm?

BTW, I do not quite get the point of setting the function name to xxx_with_host? Would you please explain? If no specific reasons, how about naming it sth like `set_host_bridge_with_nm`?

Reply:
Awesome! good idea!

Reply:
well, regarding to the func `set_host_bridge_interface_with_host()`  let me do the explanation here. 
1stly, just compare both with config_host_and_kvm_container() and cleanup_host_and_kvm_container(),  `set_host_bridge_interface_with_host()` just only set up host bridge network interface on alp host without under kvm container.

Reply:
Short update, had updated func name as `set_host_bridge_interface_with_nm()` and move it to  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/virt_autotest/utils.pm